packages/components/managed-data-table/src/data-table-settings/data-table-settings.spec.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { screen, render, fireEvent } from '../../../../../test/test-utils';
+import DataTableSettings, {
+  getDropdownOptions,
+  getMappedColumns,
+  getSelectedColumns,
```

Comment:
I don't see why we need to expose those internal functions...it should be possible with RTL to test the component as it is.